gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# Remove .log Files

### Description

Git no longer tracks `.log` files in `showup/management/commands/logs/`. However the directory `logs/` still exists.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
```

Once you run that last command, you should see no errors on the console. You can go on to check `/events` once you're logged in and see the events appear as they should.
